### PR TITLE
docs-fr | concepts | services-networking | dns

### PR DESCRIPTION
**Pull request by awaddiz**

### CODE REVIEW

**Comment by sieben** [comment]
@awaddiz Est ce que tu peux nettoyer tes commits ? Je te conseille de faire un rebase au dessus de la branche master

**Comment by awaddiz** [comment]
I signed it

File: content/fr/docs/concepts/services-networking/dns-pod-service.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+{{% capture body %}}
+
+## Introduction
+
+Kubernetes planifie un pod et un service DNS sur le cluster et configure
+les kubelets pour indiquer à chaque conteneur d'utiliser l'adresse IP du service DNS pour résoudre les noms DNS.
+
+### Quels composants obtiennent des noms DNS?
+
+Chaque service défini dans le cluster (y compris le serveur DNS lui-même) a nom DNS. Par défaut, la liste de recherche DNS du client d'un pod
```
Comment (feloy) [inline content/fr/docs/concepts/services-networking/dns-pod-service.md]: a un nom dns

File: content/fr/docs/concepts/services-networking/dns-pod-service.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+Les sections suivantes détaillent les types d’enregistrement et la structure supportée par Kubernetes. Toute autre structure ou noms ou requêtes qui fonctionnent sont
+considérés comme des détails d'implémentation et peuvent changer sans préavis.
+Pour une spécification plus à jour, voir
+[Découverte des services basée sur le DNS Kubernetes](https://github.com/kubernetes/dns/blob/master/docs/specification.md).
+
+## Services
+
+### Enregistrement A
+
+Les services "normaux" (pas sans en-tête) se voient attribuer un enregistrement DNS A, et a un nom sous forme : `mon-service.mon-namespace.svc.cluster.local`. La résolution de ce nom donne l'adresse `ClusterIP` du service.
```
Comment (feloy) [inline content/fr/docs/concepts/services-networking/dns-pod-service.md]: et ont un nom

Comment (feloy) [reply content/fr/docs/concepts/services-networking/dns-pod-service.md]: sous la forme

File: content/fr/docs/concepts/services-networking/dns-pod-service.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+Les Services "Headless" (ou sans en-tête, c'est à dire sans ClusterIP) aura également un enregistrement type A, donc un nom sous la forme : `mon-service.mon-namespace.svc.cluster.local`. Contrairement aux Services Normaux, cela résout l'ensemble des adresses IP des pods sélectionnés par le Service.
```
Comment (feloy) [inline content/fr/docs/concepts/services-networking/dns-pod-service.md]: auront également

File: content/fr/docs/concepts/services-networking/dns-pod-service.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+Lorsque cette option est activée, un enregistrement DNS A est attribué aux pods sous la forme `adresse-ip-du-pod.mon-namespace.pod.cluster.local`.
+
+Par exemple, un pod avec l’IP `1.2.3.4` dans le namespace (espace de nommage) `default` avec un nom DNS de `cluster.local` aurait une entrée : `1-2-3-4.default.pod.cluster.local`.
+
+### Nom d'hôte et sous-domaine d'un pod
+
+Actuellement, lorsqu'un pod est créé, son nom d'hôte a la valeur `metadata.name` du pod.
+
+La spécification du pod a un champ optionnel `hostname`, qui peut être utiliser pour spécifier la valeur du nom d'hôte du pod. Quand c'est spécifié, ce dernier a la priorité sur le nom du pod. Par exemple, si un pod avec un `hostname` qui a comme valeur "`mon-hôte`", son nom d'hôte sera "`mon-hôte`".
```
Comment (feloy) [inline content/fr/docs/concepts/services-networking/dns-pod-service.md]: qui peut être utilisé

Comment (feloy) [reply content/fr/docs/concepts/services-networking/dns-pod-service.md]: Dans la dernière phrase, il manque le verbe principal. Je désirais plutôt
Par exemple, si un pod a un hostname ayant la valeur mon-hote, son nom d'hôte sera mon-hote

File: content/fr/docs/concepts/services-networking/dns-pod-service.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+La spécification du pod a également un champ optionnel `subdomain` qui peut être utiliser pour spécifier son sous-domaine. Par exemple, un pod avec une valeur "`foo`" du champ `hostname` et une valeur "` bar`" du champ `subdomain`, dans le namespace "`mon-namespace`", aura un nom de domaine (FQDN) "`foo.bar.mon-namespace.svc.cluster.local`".
```
Comment (feloy) [inline content/fr/docs/concepts/services-networking/dns-pod-service.md]: qui peut être utilisé

File: content/fr/docs/concepts/services-networking/dns-pod-service.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
+---
+reviewers:
```
Comment (remyleone) [inline content/fr/docs/concepts/services-networking/dns-pod-service.md]: You can remove the reviewers that will not speak French

**Comment by zacharysarah** [comment]
It looks like this PR is caught in limbo after merging https://github.com/kubernetes/website/pull/13598 and removing Travis from our CI pipeline. (Let the joyous news be spread!) 

@remyleone Would you please add a trivial commit to trigger a new build? If that doesn't work, the next step is to close this PR and cherry-pick the commits into a new PR.

**Comment by remyleone** [comment]
@awaddiz Could you rebase on master and push again?

**Comment by zacharysarah** [comment]
@awaddiz 👋Thanks for rebasing. Unfortunately, it looks like this PR is still stuck in the old CI. I'm sorry to ask it, but could you please squash your commits, then `git cherry-pick` the squashed commit to a new PR? Thanks in advance! 🙇

**Comment by remyleone** [comment]
https://github.com/kubernetes/website/pull/13765

**Pull request closed without merging**